docs/template_plugin/tests/functional/op_reference/gather_tree.cpp

```diff
@@ -3,91 +3,171 @@
 //
 
 #include <gtest/gtest.h>
+#include <random>
 
-#include "openvino/op/gather_tree.hpp"
+#include "openvino/opsets/opset1.hpp"
 #include "base_reference_test.hpp"
+#include "ngraph/runtime/reference/gather_tree.hpp"
+#include "ngraph_functions/utils/data_utils.hpp"
```

Comment:
Why do we need these headers?

Reply:
Updated in https://github.com/openvinotoolkit/openvino/pull/9141
Thanks for your comment